source/jormungandr/tests/tests_mechanism.py

```diff
@@ -100,9 +100,9 @@ def kill_all_krakens(cls):
     def create_dummy_json(cls):
         conf_template_str = ('{{ \n'
                              '    "key": "{instance_name}",\n'
-                             '    "zmq_socket": "ipc:///tmp/{instance_name}",\n'
+                             '    "zmq_socket": "ipc://%s/{instance_name}",\n'
                              '    "realtime_proxies": {proxy_conf}\n'
-                             '}}')
+                             '}}' % (os.environ.get('WORKSPACE') or os.environ.get('HOME')))
```

Comment:
I'd prefer `/tmp` to directly in my home.

I think something like `(os.environ.get('KRAKEN_BUILD_DIR') or '/tmp')` make more sense as when lanched by nose, `KRAKEN_BUILD_DIR` must be defined to launch the mocks.